queue/limited_serializable.go

```diff
@@ -147,6 +147,17 @@ func (q *limitedSizeSerializableLocal) Put(ctx context.Context, j amboy.Job) err
 	return nil
 }
 
+// PutMany adds jobs to the queue, returning an error if the queue is not yet
+// opened or if the job already exists in the queue. If the queue is at
+// capacity, PutMany will fail.
+func (q *limitedSizeSerializableLocal) PutMany(ctx context.Context, jobs []amboy.Job) error {
+	catcher := grip.NewBasicCatcher()
+	for _, j := range jobs {
+		catcher.Add(q.Put(ctx, j))
```

Comment:
Same thing as above - include job ID in the wrapped error.

Reply:
done